.pre-commit-hooks.yaml

```diff
@@ -50,7 +50,7 @@
 -   id: rst-backticks
     name: rst ``code`` is two backticks
     description: 'Detect common mistake of using single backticks when writing rst'
-    entry: '(^| )`[^`]+`([^_]|$)'
+    entry: '^(?![ >]).*(^| )`[^`]+`([^_]|$)'
```

Comment:
this false-positives for

```
some text
 where someone `did this`
```

I think the case for this is exceedingly rare -- I don't think we should change the regex to accommodate this edge case

Reply:
All the misflagged code block issues in https://github.com/python/peps/pull/1668/files began with four-space indents.

Shall I switch it to only skip for `^` + 4 spaces?

Reply:
yeah that seems fine

Reply:
Thanks, updated/rebased!